.github/workflows/pull_request.yml

```diff
@@ -5,36 +5,58 @@ name: Pull Request
 
 on:
   pull_request:
-    branches: [ master ]
+    branches: [ master, prerelease ]
```

Comment:
Remove `prerelease` here. It was added to run CI on our fork branch, but it shouldn't go to their master